src/app/page.tsx

```diff
@@ -1,113 +1,38 @@
 
 export default function Home() {
+    const [images, setImages] = useState<File[]>([]);
+  
+    const handleImageSelect = (e: ChangeEvent<HTMLInputElement>) => {
+      const selectedImages = Array.from(e.target.files as FileList);
+      setImages(images.concat(selectedImages));
+    };
+  
+    const handleImageDelete = (index: number) => {
+      const updatedImages = images.slice();
+      updatedImages.splice(index, 1);
+      setImages(updatedImages);
+    };
+
   return (
     <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      <div className="z-10 max-w-5xl w-full items-center justify-between font-mono text-sm lg:flex">
-        <p className="fixed left-0 top-0 flex w-full justify-center border-b border-gray-300 bg-gradient-to-b from-zinc-200 pb-6 pt-8 backdrop-blur-2xl dark:border-neutral-800 dark:bg-zinc-800/30 dark:from-inherit lg:static lg:w-auto  lg:rounded-xl lg:border lg:bg-gray-200 lg:p-4 lg:dark:bg-zinc-800/30">
-          Get started by editing&nbsp;
-          <code className="font-mono font-bold">src/app/page.tsx</code>
-        </p>
-        <div className="fixed bottom-0 left-0 flex h-48 w-full items-end justify-center bg-gradient-to-t from-white via-white dark:from-black dark:via-black lg:static lg:h-auto lg:w-auto lg:bg-none">
-          <a
-            className="pointer-events-none flex place-items-center gap-2 p-8 lg:pointer-events-auto lg:p-0"
-            href="https://vercel.com?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            By{' '}
-            <Image
-              src="/vercel.svg"
-              alt="Vercel Logo"
-              className="dark:invert"
-              width={100}
-              height={24}
-              priority
-            />
-          </a>
-        </div>
-      </div>
-
-      <div className="relative flex place-items-center before:absolute before:h-[300px] before:w-[480px] before:-translate-x-1/2 before:rounded-full before:bg-gradient-radial before:from-white before:to-transparent before:blur-2xl before:content-[''] after:absolute after:-z-20 after:h-[180px] after:w-[240px] after:translate-x-1/3 after:bg-gradient-conic after:from-sky-200 after:via-blue-200 after:blur-2xl after:content-[''] before:dark:bg-gradient-to-br before:dark:from-transparent before:dark:to-blue-700 before:dark:opacity-10 after:dark:from-sky-900 after:dark:via-[#0141ff] after:dark:opacity-40 before:lg:h-[360px] z-[-1]">
-        <Image
-          className="relative dark:drop-shadow-[0_0_0.3rem_#ffffff70] dark:invert"
-          src="/next.svg"
-          alt="Next.js Logo"
-          width={180}
-          height={37}
-          priority
-        />
-      </div>
+      <h1>HDRICalibrationTool</h1>
+      <div>
+        <h2>Image Upload</h2>
+        <input type="file" accept="image/*" multiple onChange={handleImageSelect} />
```

Comment:
So I have this annoying browser default that displays a count of files selected (not the current number of files in the array). We could use `useRef` to replace that input by a button:

```tsx
<input type="file" accept="image/*" multiple onChange={handleImageSelect} ref={fileInputRef} className="hidden"/>
<button onClick={handleClick} className="btn">Select Files</button>
```

You need to define the following on top:
```tsx
const fileInputRef = useRef(null);
const handleClick = () => {
      fileInputRef.current.click();
};
```
There might be a better way to do this but it should work!

Reply:
Might need to style that button, though! 😅

Reply:
Could we also re-add the correct length of the images array?
Somethign like: `<div>Image Count: {images.length}</div>`